stacks-signer/src/config.rs

```diff
@@ -192,6 +189,8 @@ struct RawConfigFile {
     pub nonce_timeout_ms: Option<u64>,
     /// timeout in (millisecs) to gather signature shares
     pub sign_timeout_ms: Option<u64>,
+    /// the STX tx fee to use in uSTX
+    pub tx_fee_ms: Option<u64>,
```

Comment:
What's the difference between `tx_fee` and `tx_fee_ms`?

Reply:
tx_fee ms is just in microSTX. I put it in the config file that a user interacts with as such to make it clear what they are denoating. Drop the ms elsewhere but can do consistently if better.